nextflow.config

```diff
@@ -11,28 +11,29 @@ params {
 
     // TODO nf-core: Specify your pipeline's command line flags
     // Input options
-    input                      = null
+    input                      = '/analyses/users/nokuzothan/Virolocate/sra_samplesheet.csv'
 
     // Output options
     diamond_output_format = 'tsv'
 
     //file paths
     outdir="results"
-    // trimmomatic_adapters = 'null'
-    // samplesheet = 'null'
-
-
+    trimmomatic_adapters = '/analyses/users/nokuzothan/Virolocate/trim_adapters.fa'
+    samplesheet = '/analyses/users/nokuzothan/Virolocate/sra_samplesheet.csv'
+    taxidlist = 'stub_test_data/taxidlist.txt'
 
     // //databases
-    // rvdb_fasta = "null"
+    // rvdb_fasta = "/home/nokuzothan/workspace/disc_pipe/ncbidb/RVDB/v30.0/U-RVDBv30.0-prot.fasta"
     // viral_csv = "null"
     // ncbi_fasta = "null"
     // ncbi_nr_fasta = "null"
-    // ncbi_nt_fasta = "null"
-    // taxonkit_db = "null"
-    // blastx_nr_fasta = "null"
-    // blastn_db = "null"
-    // final_blastx_db = "null"
+    rvdb_fasta = "/analyses/users/nokuzothan/disc_pipe/ncbidb/fasta/nr.faa"
+    viral_csv = "/analyses/download/Virolocate/virus_taxonomy_lvls.csv"
+    ncbi_nr_fasta = "/analyses/users/nokuzothan/disc_pipe/ncbidb/fasta/nr.faa"
+    ncbi_nt_db = "/analyses/users/nokuzothan/disc_pipe/ncbidb/nt"
+    taxdb = "/analyses/users/nokuzothan/ncbi_taxdump"
```

Comment:
In the `nextflow.config` file, these values shouldn't be hard-coded as they are relevant only to your environment. 

The best-practice is to simply assign `null` or `[]` for these parameters in the `nextflow.config`